changelogs/6.8.asciidoc

```diff
@@ -0,0 +1,16 @@
+[[release-notes-6.8]]
+== APM Server version 6.8
+
+https://github.com/elastic/apm-server/compare/6.7\...6.8[View commits]
```

Comment:
```diff
- https://github.com/elastic/apm-server/compare/6.7\...6.8[View commits]
+ https://github.com/elastic/apm-server/compare/6.7...6.8[View commits]
```

Reply:
The `\` is necessary, otherwise it will be parsed as an ellipsis.
Check for instance links in https://github.com/elastic/apm-server/blob/master/changelogs/7.1.asciidoc (they are broken)
I'll fix those now

Reply:
The `\` is needed for links to work in markdown

Reply:
Ahhhh I missed this message from you. Thanks @jalvz